app/src/org/commcare/views/QuestionsView.java

```diff
@@ -65,12 +66,14 @@
 
     private SpannableStringBuilder mGroupLabel;
 
+    private BlockingActionsManager blockingActionsManager;
+
     /**
      * If enabled, we use dividers between question prompts
      */
     private static final boolean SEPERATORS_ENABLED = false;
 
-    public QuestionsView(Context context) {
+    public QuestionsView(Context context, BlockingActionsManager blockingActionsManager) {
```

Comment:
Sorry, not following. The `BlockingActionsManager` has to originate in the `FormEntryActivityUIController` because that is where we check it. So I just followed the natural path that got me from `FormEntryActivityUIController` --> `WidgetFactory.createWidgetFromPrompt`, which seems to require going through `QuestionsView`, since that is the link between them